VigilanceMeteoFrance/VigilanceMeteoFrance/Program.cs

```diff
@@ -0,0 +1,157 @@
+﻿using Constellation;
+using Constellation.Package;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace VigilanceMeteoFrance
+{
+    public class Program : PackageBase
+    {
+        /// <summary>
+        /// Refresh rate of the vigilance.
+        /// </summary>
+        public int RefreshInterval { get; set; }
+
+        /// <summary>
+        /// Departements' number to watch.
+        /// </summary>
+        public string Departement { get; set; }
+
+        static void Main(string[] args)
+        {
+            PackageHost.Start<Program>(args);
+        }
+
+        public override void OnStart()
+        {
+            PackageHost.WriteInfo("Package starting - IsRunning: {0} - IsConnected: {1}", PackageHost.IsRunning, PackageHost.IsConnected);
+            this.RefreshInterval = PackageHost.GetSettingValue<int>("RefreshInterval");
+            this.Departement = PackageHost.GetSettingValue<string>("Departement");
+            if (Departement!="00")
+            {
+                Task.Factory.StartNew(() =>
```

Comment:
Avoid launching long running task, prefer to launch a new task every x minutes.

If your task failed one time, it doesn't be reloaded.